jetson/nav/search/searchStateMachine.cpp

```diff
@@ -150,6 +151,20 @@ NavState SearchStateMachine::executeDriveToTarget() {
     return NavState::DriveToTarget;
 } // executeDriveToTarget()
 
+// Sends search path rover takes when trying to find posts
+void SearchStateMachine::publishSearchPoints() {
+    // Construct vector from deque
+    std::vector<Odometry> arr(mSearchPoints.begin(), mSearchPoints.end());
+    SearchPoints searchPoints{
+        .search_pattern_size  = (int32_t)mSearchPoints.size(),
```

Comment:
in general kind of a nitpick can you use ```std::static_cast<>```

Reply:
yeah lemme fix that, good call